web-app/js/portal/lang/en.js

```diff
@@ -229,7 +228,7 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     depthLabel: 'Depth:',
     elevationLabel: 'Elevation:',
 
-    removeDataCollection: 'Remove data collection',
+    removeDataCollection: 'Remove this data collection from the Subset and Download pages, the collection can be re-added from the search',
```

Comment:
I think 'removeButtonTooltip' is a better name for this dictionary entry because it tells you *how* this value is being used (in a tooltip).